presto-docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -11,13 +11,17 @@ Overview
 --------
 
 This connector allows the use of `Apache Kafka <https://kafka.apache.org/>`_
-topics as tables in Presto.
-Each message is presented as a row in Presto.
+topics as tables in Presto. Each message is presented as a row in Presto.
 
 Topics can be live. Rows appear as data arrives, and disappear as
 segments get dropped. This can result in strange behavior if accessing the
 same table multiple times in a single query (e.g., performing a self join).
 
+The connector reads message data from Kafka topics in parallel across workers to
+achieve a significant performance gain. The size of data sets for this
```

Comment:
> to achieve a significant performance gain

To me that could be written in pull request. Like "this change was introduced to to achieve a significant performance gain". In general documentation I think it should be only: "The connector reads message data from Kafka topics in parallel across workers". The reason is obvious.

Reply:
I don't think the reason is obvious to a normal user at all and I think it is important for beginner users to be told why they would want to use the connector at all including performance benefits.